interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -124,6 +143,56 @@ paths:
       description: A task ID of an initiated task.
       schema:
         $ref: '#/components/schemas/TaskID'
+  /api/flight-records-upload/json:
```

Comment:
Let's stay consistent with dashes versus underscores and use underscores since they can be used in variable names.  And, the means by which the flight records are acquired shouldn't matter, so let's change this to just `/api/flight_records/json` (and the corresponding variant below to `/api/flight_records/kml`)